5.5/docker-entrypoint.sh

```diff
@@ -99,10 +99,11 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		mysql=( mysql --protocol=socket -uroot -hlocalhost --socket="${SOCKET}" )
 
 		for i in {30..0}; do
-			if echo 'SELECT 1' | "${mysql[@]}" &> /dev/null; then
+			echo 'Trying to connect to mysql...'
+			if echo 'SELECT 1' | "${mysql[@]}" > /dev/null; then
```

Comment:
stderr can be very useful sometimes. We should not always suppress it. So here I removed the '&'.